<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-FUDE26F" ignored="0" name="QLPB_010" tests="1" failures="1" timestamp="2020-06-25T12:33:21 ICT" time="0.152" errors="0">
  <testcase name="addStaff1" time="0.03" classname="Test.DepartDaoTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Test.DepartDaoTest.addStaff1(DepartDaoTest.java:66)
at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:172)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- addStaff1 -->
</testsuite> <!-- QLPB_010 -->
